documentation/deliverables/individual_deliverables/open-source-group-kickoff.md

```diff
@@ -0,0 +1,148 @@
+# Open Source Group Kickoff
+
+| Field           | Value                                                        |
+| --------------- | ------------------------------------------------------------ |
+| Document Status | Draft                                                        |
+| Epic Link       | [Issue 72](https://github.com/HHS/grants-equity/issues/72)   |
+| Epic Dashboard  | [Milestone Roadmap](https://github.com/orgs/HHS/projects/12) |
+
+
+## Short description
+
+-**What:** Set up the tools and processes needed to build an open source community around the Simpler Grants.gov initiative
```

Comment:
Nit: for some reason the rendered formatting here for the bullets isn't working. Maybe need a space between `-` and `**`